shared/container.go

```diff
@@ -263,6 +263,10 @@ var KnownContainerConfigKeys = map[string]func(value string) error{
 	"security.syscalls.blacklist":         IsAny,
 	"security.syscalls.whitelist":         IsAny,
 
+	"snapshots.schedule":         IsAny,
+	"snapshots.schedule.stopped": IsBool,
+	"snapshots.pattern":          IsAny,
```

Comment:
This should also apply to manually created snapshots where no name is specified.